packages/accordions/src/styled/accordion/StyledPanel.ts

```diff
@@ -7,50 +7,52 @@
 
 import styled, { css, ThemeProps, DefaultTheme } from 'styled-components';
 import {
-  getColorV8,
   getLineHeight,
   retrieveComponentStyles,
-  DEFAULT_THEME
+  DEFAULT_THEME,
+  getColor
 } from '@zendeskgarden/react-theming';
 
 interface IStyledPanel {
   inert?: string;
-  isBare?: boolean;
-  isCompact?: boolean;
-  isExpanded?: boolean;
-  isAnimated?: boolean;
+  $isBare?: boolean;
+  $isCompact?: boolean;
+  $isExpanded?: boolean;
+  $isAnimated?: boolean;
 }
 
 const COMPONENT_ID = 'accordions.panel';
 
 const colorStyles = (props: IStyledPanel & ThemeProps<DefaultTheme>) => {
-  const { theme, isBare } = props;
+  const { theme, $isBare } = props;
```

Comment:
[nit] I realize this is inherited code, but we tend to prefer destructuring directly in the function arguments.